src/api/db/schema.rb

```diff
@@ -10,10 +10,10 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2022_04_13_223301) do
+ActiveRecord::Schema.define(version: 2022_05_18_094002) do
 
   create_table "architectures", id: :integer, charset: "utf8mb4", collation: "utf8mb4_unicode_ci", options: "ENGINE=InnoDB ROW_FORMAT=DYNAMIC", force: :cascade do |t|
-    t.string "name", null: false, collation: "utf8_general_ci"
+    t.string "name", null: false, collation: "utf8mb3_general_ci"
```

Comment:
Please do not include those changes here. They might stem from a mariadb update in the dev-env.